src/commands/ext-uninstall.ts

```diff
@@ -15,6 +18,26 @@ import { requirePermissions } from "../requirePermissions";
 import * as utils from "../utils";
 import * as logger from "../logger";
 
+marked.setOptions({
```

Comment:
Does this modify a global state that will affect other uses of `marked` in the codebase?  If so is there a way to get an "instance"?

Reply:
This creates a reference instance: see https://marked.js.org/#/USING_ADVANCED.md